README.md

```diff
@@ -2,29 +2,29 @@
 
 # 1x.engineer
 
-This repo is the source of [1x.engineer](http://1x.engineer/), a website illustrating what a 1x engineer _really_ is.
+This repo is the source of [1x.engineer](http://1x.engineer/), this is a website which _actually_ illustrates what a 1x engineer is.
 
 ## Contributing
 
-Want to help contribute to 1x.engineer? Cool! Feel free to PR whatever you think is appropriate and fitting with the theme, as long as your contributions are in-line with the project's [Code of Conduct](./CODE_OF_CONDUCT.md). I'll try to be on top of PRs, and will add those who get involved with the project beyond single PRs as collaborators.
+Want to help contribute to [1x.engineer](http://1x.engineer/)? Cool! Feel free to PR(_P_ull _R_equests) whatever you think is appropriate and fitting with the theme, as long as your contributions are in-line with the project's [Code of Conduct](./CODE_OF_CONDUCT.md). I'll try to be on top of PRs, and will add those who get involved with the project beyond single PRs as collaborators.
 
 ### New Traits
 
-Feel free to add new traits of 1x developers. We generally try to keep traits free of blame, and be sure not to exclude 1x developers in the phrasing or intent of the traits.
+Feel free to add new traits for 1x developers. We generally try to keep traits free of blame, and be sure not to exclude 1x developers in the phrasing or intent of the traits.
 
 ### Editing Traits
 
-If there are refinements that can be made to existing traits to make them more accurate or inclusive of 1x developers, please feel free to make them – your contributions are more than welcome.
+If there are refinements that can be made to existing traits to make them more accurate or inclusive of 1x developers, please feel free to make them – your contributions are most welcomed.
 
-Additionally, if you feel that a trait doesn't represent 1x developers, please feel free to remove it and share why. This project goes both ways.
+Additionally, if you feel that a trait doesn't represent 1x developers, please feel free to remove it and share the reason _(why?)_. This project goes both ways.
 
 ### Trolls
 
-Since this is a very human-focused project, it will attract troll Pull Requests, Issues, and comments. Please don't. If you do, we'll just block you from the repository.
+Since this is a very human-focused project, it will attract troll Pull Requests, Issues, and comments. Please don't submit troll Pull Requests. If you do, _"SORRY"_ unfortunately we'll be forced to block you from the repository.
```

Comment:
I like your clarification here, but let's keep it to the spirit of the document.

What about:
`Since this is a very human-focused project, it will attract troll Pull Requests, Issues, and comments. Please don't submit troll content in any form. If you do, we'll be forced to block you from the repository, per the [Code of Conduct](./CODE_OF_CONDUCT.md).`